src/Language/PureScript/Docs/Convert/Single.hs

```diff
@@ -200,21 +201,22 @@ convertDeclaration _ _ = Nothing
 convertComments :: [P.Comment] -> Maybe String
 convertComments cs = do
   let raw = concatMap toLines cs
-  guard (all hasPipe raw && not (null raw))
-  return (go raw)
-  where
-  go = unlines . map stripPipes
+  let docs = catMaybes (map stripPipe raw)
+  guard (not (null docs))
+  pure (unlines docs)
 
+  where
   toLines (P.LineComment s) = [s]
   toLines (P.BlockComment s) = lines s
 
-  hasPipe s = case dropWhile (== ' ') s of { ('|':_) -> True; _ -> False }
-
-  stripPipes = dropPipe . dropWhile (== ' ')
-
-  dropPipe ('|':' ':s) = s
-  dropPipe ('|':s) = s
-  dropPipe s = s
+  stripPipe s' =
```

Comment:
When I chose this name, I was thinking along the lines of stripPrefix :: String -> Maybe String, but sure, I can change it if you think takePiped is more appropriate.